PlayCover/Views/App Views/PlayAppView.swift

```diff
@@ -181,14 +180,26 @@ struct PlayAppConditionalView: View {
     @Binding var selectedTextColor: Color
     @Binding var selected: PlayApp?
 
+    @ObservedObject var app: PlayApp
+
     @State var iconURL: URL?
-    @State var app: PlayApp
     @State var isList: Bool
 
     var body: some View {
         Group {
             if isList {
                 HStack(alignment: .center, spacing: 0) {
+                    let noPlayToolsImage = Image(systemName: "exclamationmark.triangle.fill")
```

Comment:
Either make these icons `exclamationmark.triangle` or make the IPA Library download icon `arrow.down.circle.fill`